custom/bihar/reports/display.py

```diff
@@ -632,7 +633,7 @@ def dob_age(self):
                     dob = dob["#value"]
                 days = (date.today() - CaseDisplay.parse_date(self, dob).date()).days
                 dob = self.parse_date(dob)
-                return "%s, %s" % (dob, int(days/365.25))
+                return "%s, %s" % (dob, int(days / 365.25))
```

Comment:
was this supposed to be `//`?

Reply:
I think ```int(days / 365.25)``` and ```int(days // 365.25)``` should always return the same value